src/libsystemd-network/sd-dhcp-server.c

```diff
@@ -712,7 +715,54 @@ static int get_pool_offset(sd_dhcp_server *server, be32_t requested_ip) {
         return be32toh(requested_ip & ~server->netmask) - server->pool_offset;
 }
 
-static int dhcp_server_relay_message(sd_dhcp_server *server, DHCPMessage *message, size_t opt_length) {
+static size_t relay_option_length(sd_dhcp_server *server) {
+        size_t length = 0;
+        if (server->agent_circuit_id)
```

Comment:
Please add an assertion for `server`, and a blank line after the variable declaration.